inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor/bottomBar/bottomBarComponent.tsx

```diff
@@ -60,7 +116,15 @@ IBottomBarComponentState
                 {
                     this.props.context.activeAnimations.length > 0 &&
                     <div id="bottom-bar-total">
-                        {this._renderMaxFrame()}
+                        <TextInputComponent 
+                        isNumber={true}
+                        value={this.state.clipLength}
+                        tooltip="Clip Length"
+                        id="clip-range"
+                        onValueAsNumberChanged={(newValue, isFocused) => {

```

Comment:
I would recommend to also validate it when user hits enter

Reply:
Added it!